libs/semanticsearch/include/semanticsearch/schema/vocabulary_instance.hpp

```diff
@@ -36,10 +36,7 @@ class VocabularyInstance : public std::enable_shared_from_this<VocabularyInstanc
   static Vocabulary New(T data)
   {
     // TODO(private issue 143): add destructor
-    Vocabulary ret;
-    ret.reset(new VocabularyInstance(std::type_index(typeid(T)), new T(data)));
-
-    return ret;
+    return Vocabulary{new VocabularyInstance(std::type_index(typeid(T)), new T(data))};  // NOLINT
```

Comment:
Has this snuck in and is causing build failures?

Reply:
Yes it suddenly started failing the static analysis checks, not really sure why this is to be honest, but had to mark it as `NOLINT` because its suggestion (use `std::make_shared`) can't work because `VocabularyInstance` is private